README.md

```diff
@@ -28,10 +28,9 @@ Rust bindings for the NumPy C-API.
 
 ### Write a Python module in Rust
 
-Please see the [simple-extension](https://github.com/PyO3/rust-numpy/tree/main/examples/simple-extension)
-directory for the complete example.
+Please see the [simple](examples/simple) example for how to get started.
 
-Also, we have an example project with [ndarray-linalg](https://github.com/PyO3/rust-numpy/tree/main/examples/linalg).
+We also have examples using [ndarray-linalg](examples/linalg) and [rayon](examples/parallel).
```

Comment:
I know "we" didn't originate from you, though if we're editing this anyway I think it's better to remove such pronouns (google agrees https://developers.google.com/style/pronouns#personal-pronouns).

```suggestion
There are also examples using [ndarray-linalg](examples/linalg) and [rayon](examples/parallel).
```